bootstrap.sh

```diff
@@ -15,3 +15,7 @@ pipenv sync --dev
 # The app logs are redirected to the `runserver.log` file.
 (setsid pipenv run \
     python manage.py runserver 0.0.0.0:8000 > runserver.log 2>&1 &)
+
+# Migrations
+pipenv run python manage.py makemigrations tasties_app
+pipenv run python manage.py migrate
```

Comment:
Why do you need this as part of the bootstrap?

Reply:
For the first question, that's what I was trying to resolve at the beginning of the day on slack.
I'm trying to understand what you are proposing. Do you want each PR that handles a model to commit the same initial migration file ('0001_init.py')? If not, can you clarify?
Because the problem for us is that we are working on different models, in parallel. Some of the models depend on different models, so we wanted to create the same environment, as well as basic data, for all the models to work on.

As for the second question, it's required in the SOW.

Reply:
Ok, I think I didn't understand your question in slack correctly...
You can create multiple migration files so you won't override others' work - see [this](https://github.com/redhat-beyond/ResuMe/tree/main/posts/migrations) as an example.

Yes, you are correct this is part of the SOW I forgot... but you only need the migrate command as you already created the migration files so no need to run it every time the VM comes up

Reply:
Alright, so if I understand correctly, we can close this PR, and first merge Ohad's #43, because he committed this migrations file.
After that, all other models should be ok.